course_info/templates/course_info/partials/details.html

```diff
@@ -0,0 +1,211 @@
+{% verbatim %}
+
+<nav>
```

Comment:
So can the course instance list data be referenced in the same scope/passed into the `DetailsController`? That would be great. I can imagine a nice pattern where most of the fields are then immediately available, we show a loading/updating indicator, and we call the details page (and people page) in the background to ensure we have up-to-date information, and finalize the form data once we have those pieces. Every field group could in fact have its own loading indicator predicated upon whether its data point is `null` or `undefined`, and be filled in as need be. Agree that's something to iterate on in 2376 though.